async_jobs.go

```diff
@@ -21,6 +21,48 @@ type AsyncJobResult struct {
 	UserID          *UUID            `json:"userid,omitempty" doc:"the user that executed the async command"`
 }
 
+// DeepCopy create a true copy of the receiver.
+func (a *AsyncJobResult) DeepCopy() *AsyncJobResult {
+	if a == nil {
+		return nil
+	}
+
+	return &AsyncJobResult{
+		AccountID:       a.AccountID.DeepCopy(),
+		Cmd:             a.Cmd,
+		Created:         a.Created,
+		JobID:           a.JobID.DeepCopy(),
+		JobInstanceID:   a.JobInstanceID.DeepCopy(),
+		JobInstanceType: a.JobInstanceType,
+		JobProcStatus:   a.JobProcStatus,
+		JobResult:       a.JobResult,
+		JobResultCode:   a.JobResultCode,
+		JobResultType:   a.JobResultType,
+		JobStatus:       a.JobStatus,
+		UserID:          a.UserID.DeepCopy(),
+	}
+}
+
+// DeepCopyInto copies the receiver into out.
+//
+// In (a) must be non nil. out must be non nil
+func (a *AsyncJobResult) DeepCopyInto(out *AsyncJobResult) {
+	*out = AsyncJobResult{
```

Comment:
Why not use `a.DeepCopy() here ?

Reply:
`// In (a) must be non nil. out must be non nil`
we want that (program crash), not nil result